src/bridge/memory_util/openssl_structs_test.cc

```diff
@@ -27,19 +27,31 @@ namespace {
 using ::testing::Not;
 using ::testing::IsNull;
 
-TEST(OpenSSLMakeTest, MakeRsaSetsDeleter) {
+TEST(OpenSSLMakeTest, MakeEngineSetsDeleter) {
```

Comment:
OpenSsl?

Reply:
Changed because of https://github.com/googleinterns/cloud-kms-oss-tools/pull/44#discussion_r450473735

Reply:
Oh whoops, I misunderstood this comment. I'll fix this